### PR TITLE
ability to just use R instead of Rscript 

### PR DESCRIPTION
**Pull request by muschellij2**
Allowed to use R instead of Rscript so that things print out.

### CODE REVIEW

**Comment by MarkEdmondson1234** [comment]
Cool, don't quite grok when and where you should use Rscript vs R

File: R/buildsteps_templates.R [hunk 1]
```diff
@@ -510,6 +512,8 @@ cr_buildstep_r <- function(r,
      name <- basename(name)
   }
 
+  r_cmd = match.arg(r_cmd)
```
Comment (MarkEdmondson1234) [inline R/buildsteps_templates.R:515]: This should be <-  :)

**Comment by muschellij2** [comment]
Rscript - nothing is printed out in GCP - aka no "log"
R - header and everything prints out like:
```
R version 4.1.1 (2021-08-10) -- "Kick Things"
Copyright (C) 2021 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

```

File: R/buildsteps_templates.R (hunk 1 above)
Comment (muschellij2) [reply R/buildsteps_templates.R:515]: Ahh! I changed now - next PR will have it.

**Pull request merged**